R/metrics-quantile.R

```diff
@@ -180,10 +180,31 @@ wis <- function(observed,
                 separate_results = FALSE,
                 weigh = TRUE,
                 count_median_twice = FALSE,
-                na.rm = TRUE) {
+                na.rm = FALSE) {
   assert_input_quantile(observed, predicted, quantile_level)
   reformatted <- quantile_to_interval(observed, predicted, quantile_level)
 
+  # check that all quantile levels form valid prediction intervals
+  interval_ranges <- get_range_from_quantile(
+    quantile_level[quantile_level != 0.5]
+  )
+  complete_intervals <-
+    duplicated(interval_ranges) | duplicated(interval_ranges, fromLast = TRUE)
+  if (any(!complete_intervals) && !na.rm) {
```

Comment:
on top oif the linting I think isTRUE wrapping around. logical is good practice (i.e `!isTRUE(na.rm)`)